database/pgx/pgx_test.go

```diff
@@ -318,6 +318,65 @@ func TestWithSchema(t *testing.T) {
 	})
 }
 
+func TestMigrationTableOption(t *testing.T) {
+	dktesting.ParallelTest(t, specs, func(t *testing.T, c dktest.ContainerInfo) {
+		ip, port, err := c.FirstPort()
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		addr := pgConnectionString(ip, port)
+		p := &Postgres{}
+		d, _ := p.Open(addr)
+		defer func() {
+			if err := d.Close(); err != nil {
+				t.Fatal(err)
+			}
+		}()
+
+		// create gomigrate schema
```

Comment:
This package is called `migrate` so let's not introduce a new term. e.g. name the schema `migrate`